client/src/components/pages/LogIn/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import signUpValidation from '../utils/schema';
+import Input from '../../common/Input';
+import Button from '../../common/Button';
+
+import('./index.css');
+
+class LogIn extends Component {
+  state = {
+    username: '',
+    password: '',
+    error: false,
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const { username, password } = this.state;
+    signUpValidation.isValid({ username, password }).then(res => {
+      this.setState({
+        error: !!res,
```

Comment:
check it by if ... isvaild returns boolean not string

Reply:
using if is unnecessary here, it only bloats the code and makes it harder to read. The above achieves the same  in half the code (the !! can be potentially removed since isValid returns boolean)